Dte.Common/Exceptions/Common/ErrorCode.cs

```diff
@@ -4,6 +4,18 @@ public class ErrorCode
     {
         // Http codes
         public const string InternalServerError = "InternalServerError";
+        public const string BadRequest = "Bad_Request";
```

Comment:
I'm wary of this pattern as we are duplicating the behaviour of the existing HTTP error codes, but it is consistent which how the code currently works.